flink-runtime/src/main/java/org/apache/flink/runtime/rest/messages/util/stats/StatsSummaryDto.java

```diff
@@ -16,10 +16,10 @@
  * limitations under the License.
  */
 
-package org.apache.flink.runtime.rest.messages.checkpoints;
+package org.apache.flink.runtime.rest.messages.util.stats;
```

Comment:
Renaming this may change the URN in the REST API documentation, similar to what happened in https://issues.apache.org/jira/browse/FLINK-13949. I think we should regenerate the relevant REST API documentation to reflect these updates.

Reply:
It is a good catch you raised !
Thank you .
I made the related change.  PTAL~